### DISCUSSION TITLE
Update older version of mongoose in mongodb

### DISCUSSION THREAD

**Discussion by donoftime2018** [Automattic/mongoose#14045] [General]
I'm new to open source contribution, and I want to solve a fix for Issue #14041. How can I update mongodb/mongoose source code from mongodb version 6.0.11?

**Comment by vkarpov15** [comment]
#14041 says the MongoDB server version is 6.0.11, that version doesn't have much to do with Mongoose. If you have a fix for #14041, you should just make a PR against master.
